plugins/sid/sid_instrument.cpp

```diff
@@ -637,10 +622,6 @@ sidInstrumentView::sidInstrumentView( Instrument * _instrument,
 		sync_btn->setInactiveGraphic(
 			PLUGIN_NAME::getIconPixmap( "sync" ) );
 		ToolTip::add( sync_btn, tr( "Sync" ) );
-		sync_btn->setWhatsThis( tr ( "Sync synchronizes the fundamental "
-			"frequency of Oscillator %1 with the fundamental frequency of "
-			"Oscillator %2 producing \"Hard Sync\" effects." ).arg( i+1 )
-			.arg( syncRingWidth[i] ) );
```

Comment:
Now that we aren't translating `syncRingWidth`, it's throwing an used variable exception.  It should be removed with this PR.

Reply:
Dona via e385c11. Sorry for the late reply.